main.js

```diff
@@ -0,0 +1,96 @@
+// Initiate Variables
+let positions = document.getElementsByClassName("cell"),
+  startBtn = document.getElementById("initGame"),
+  game = {};
+
+// Adds an event listener to the begin button. Then turns on intialize function.
+startBtn.addEventListener("click", () => {
+  initialize();
+});
+
+function initialize() {
+  // Initialization function. Triggered when user clicks begin button.
+  game.started = true;
+  game.player = "human";
```

Comment:
I would like to see these keys when you define the original `game` object even if they are set to `null` initially. It could introduce bugs if you add properties like this because you could assume the properties are there when they're not, and it's good to get into the habit of doing it.